includes/controllers/class-attribute.php

```diff
@@ -156,7 +156,7 @@ function( $name ) {
 			hivepress()->attribute->get_models()
 		);
 
-		if ( ! in_array( $model_name, $model_names ) || ! in_array( $meta_box, [ $model_name . '_edit', $model_name . '_search' ] ) ) {
+		if ( ! in_array( $model_name, array_merge( [ 'listing' ], $model_names ) ) || ! in_array( $meta_box, [ 'listing_attributes', $model_name . '_edit', $model_name . '_search' ] ) ) {
```

Comment:
We need this functionality for any attribute-enabled models, please check the rest of the code and avoid hard-coding the listing model name. I don't remember if Attribute component has a way to get a list of models from the outside (e.g. a helper get_models), if it's available please use it, if not we need to add one